### PR TITLE
Nc import letter file 1 print file work

### PR DESCRIPTION
**Pull request by hbrace**
# Checklist
Reminder: If any changes have been released in the [census-rm-sample-loader](https://github.com/ONSdigital/census-rm-sample-loader) or [census-rm-toolbox](https://github.com/ONSdigital/census-rm-toolbox/) then the version pins in the Pipfile need to be updated.
* [ ] Updated census-rm-toolbox version pin (if applicable)
* [ ] Updated census-rm-sample-loader version pin (if applicable)

# Blocked until after Lukes ATs PR has gone in

# Motivation and Context

There is a new action rule for non compliance print files.
# What has changed

Test added to create print files for non compliance for both England and Wales types.
# How to test?

Run ATs with PRs off attached ticket.
# Links

https://trello.com/c/r6x95aJS/1405-nc-import-letter-file-1-print-file-work-8
# Screenshots (if appropriate):

### CODE REVIEW

**Comment by hbrace** [comment]
[NC - Import letter file 1 - print file work (8)](https://trello.com/c/r6x95aJS/1405-nc-import-letter-file-1-print-file-work-8)

**Review by NickGrantX — CHANGES_REQUESTED** [review]
Needs an AT to check that a FIELD action plan will send FWMT-G CREATE messages for NCF cases

File: acceptance_tests/features/steps/action_rules.py [hunk 1]
```diff
@@ -59,3 +59,11 @@ def set_up_lsoa_driven_action_rule(context, action_type, lsoas):
 @step('the address frame delta initial contact action rule of type "{action_type}" is set')
 def set_address_delta_action_rule(context, action_type):
     setup_address_frame_delta_action_rule(context, action_type)
+
+
+@step('we schedule an action rule of type "{action_type}" with NCL and "{region}" classifiers')
+def setup_print_action_rule_once_case_action_is_drained_for_non_compliance(context, action_type, region):
+    build_and_create_action_rule(context,
+                                 DEFAULT_CLASSIFIERS + f"metadata->>'nonCompliance' = 'NCL' AND SUBSTRING(region, 1, 1) \
```
Comment (liamtoozer) [inline acceptance_tests/features/steps/action_rules.py]: The spec says this is only for `HH` case types, but the `DEFAULT_CLASSIFIERS` used for this test will include: `SPG`, `CE`, and `HH` cases. I know the sample loaded was for `HH` cases but it's probably a good idea to be explicit in what the action rule classifiers should be.

https://github.com/ONSdigital/census-rm-acceptance-tests/blob/6e73b78ee2d1bf6365d12d46b4ec843502a9ff06/acceptance_tests/utilities/mappings.py#L13

**Review by liamtoozer — APPROVED** [review]

**Review by NickGrantX — APPROVED** [review]
Looking good

**Pull request merged**